File: src/main/java/com/github/thaynarasilvapinto/SimuladorBanco/repositories/ContaRepository.java
package com.github.thaynarasilvapinto.SimuladorBanco.repositories;

import com.github.thaynarasilvapinto.SimuladorBanco.domain.Conta;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface ContaRepository extends JpaRepository<Conta, Integer> {
}

File: src/main/kotlin/com/github/thaynarasilvapinto/simuladorbanco/repositories/ContaRepository.kt
package com.github.thaynarasilvapinto.simuladorbanco.repositories

import com.github.thaynarasilvapinto.simuladorbanco.domain.Conta
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface ContaRepository : JpaRepository<Conta, Int>